packages/atomic/cypress/integration/result-list/result-components/result-multi-value-text-selectors.ts

```diff
@@ -0,0 +1,20 @@
+import {ResultListSelectors} from '../result-list-selectors';
+
+export const resultMultiValueTextComponent = 'atomic-result-multi-value-text';
+
+export const ResultMultiValueTextSelectors = {
+  value: () =>
+    ResultMultiValueTextSelectors.firstInResult()
+      .shadow()
+      .find('[part="result-multi-value-text-value"] slot'),
+  separator: () =>
+    ResultMultiValueTextSelectors.firstInResult()
+      .shadow()
+      .find('[part="result-multi-value-text-separator"]'),
+  more: () =>
```

Comment:
can we have more proper name, well, I might not sure More here is text/button/link?

Reply:
Good point, I'll change `more` to `moreLabel`